### DISCUSSION TITLE
"슬랙에 github 알림 추가하기" task 문제되는 상황

### DISCUSSION THREAD

**Discussion by limhyoseok** [DeveloperAcademy-POSTECH/toGather#111] [General]
슬랙에 github 알림을 추가하기 위해서는 레포 권한을 얻어야하는데 아카데미 소속이라서 마음대로 못하는 상황

슬랙에 깃 추가가 필요한가?에 대해 근본적인 논의 필요
- 이슈, pr 디스커션 등 알림 뜨면 편하기는 하지만 실제로 잘 보지는 않음. 메일로 알람이 오기도 하고...

필요하다고 생각되면... 방법은 아래 두개

1. 필요하다고 생각하면 아카데미에 부탁하기
2. 아카데미에 종속되어있는 레포를 분리하기

의견 남겨주세요

**Comment by yudonlee** [comment]
일단, 진행하면서 아이작한테 여쭤볼게요! 뭔가 우리앱이 오래갈거라면, 편하게 슬랙내에서 할 수 있다면 더 효율적이라는 생각이 들어서요!
